encodings/datetime-parts/src/array.rs

```diff
@@ -160,4 +161,17 @@ impl AcceptArrayVisitor for DateTimePartsArray {
     }
 }
 
-impl ArrayStatisticsCompute for DateTimePartsArray {}
+impl ArrayStatisticsCompute for DateTimePartsArray {
+    fn compute_statistics(&self, stat: Stat) -> VortexResult<StatsSet> {
+        let maybe_stat = match stat {
+            Stat::NullCount => Some(Scalar::from(self.validity().null_count(self.len())?)),
+            _ => None,
+        };
+
+        let mut stats = StatsSet::new();
+        if let Some(value) = maybe_stat {
+            stats.set(stat, value);
+        }
+        Ok(stats)
```

Comment:
I think this is a weird way to do it. If there's only one supported for now simplify?